x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.ts

```diff
@@ -47,13 +47,10 @@ export const buildBulkBody = (
     ...buildSignal([mergedDoc], rule, reason),
     ...additionalSignalFields(mergedDoc),
   };
-  const event = buildEventTypeSignal(mergedDoc);
-  // Filter out any kibana.* fields from the generated signal - kibana.* fields are aliases
-  // in siem-signals so we can't write to them, but for signals-on-signals they'll be returned
-  // in the fields API response and merged into the mergedDoc source
-  const { threshold_result: thresholdResult, kibana, ...filteredSource } = mergedDoc._source || {
+  const { threshold_result: thresholdResult, ...filteredSource } = mergedDoc._source || {
```

Comment:
This file should stay as it was. The tests are gonna fail again `ciGroup11` and this is why. You can just undo revert it back to 

```
  const event = buildEventTypeSignal(mergedDoc);
  // Filter out any kibana.* fields from the generated signal - kibana.* fields are aliases
  // in siem-signals so we can't write to them, but for signals-on-signals they'll be returned
  // in the fields API response and merged into the mergedDoc source
  const { threshold_result: thresholdResult, kibana, ...filteredSource } = mergedDoc._source || {
    threshold_result: null,
  };
```